--- a/qrconfig.php	2013-01-23 14:55:45.858052303 +0100
+++ b/qrconfig.php	2013-01-23 14:55:57.377754931 +0100
@@ -6,8 +6,8 @@
  */
      
     define('QR_CACHEABLE', true);                                                               // use cache - more disk reads but less CPU power, masks and format templates are stored there
-    define('QR_CACHE_DIR', dirname(__FILE__).DIRECTORY_SEPARATOR.'cache'.DIRECTORY_SEPARATOR);  // used when QR_CACHEABLE === true
-    define('QR_LOG_DIR', dirname(__FILE__).DIRECTORY_SEPARATOR);                                // default error logs dir   
+    define('QR_CACHE_DIR', '/var/cache/php/phpqrcode/');                                        // used when QR_CACHEABLE === true
+    define('QR_LOG_DIR', false);                                                                // default error logs dir   
     
     define('QR_FIND_BEST_MASK', true);                                                          // if true, estimates best mask (spec. default, but extremally slow; set to false to significant performance boost but (propably) worst quality code
     define('QR_FIND_FROM_RANDOM', false);                                                       // if false, checks all masks available, otherwise value tells count of masks need to be checked, mask id are got randomly
--- php-phpqrcode-1.1.4/phpqrcode.php~	2014-03-29 13:25:52.000000000 +0200
+++ php-phpqrcode-1.1.4/phpqrcode.php	2014-03-29 13:28:26.025533866 +0200
@@ -114,8 +114,8 @@
  * Config file, tuned-up for merged verion
  */
      
-    define('QR_CACHEABLE', false);       // use cache - more disk reads but less CPU power, masks and format templates are stored there
-    define('QR_CACHE_DIR', false);       // used when QR_CACHEABLE === true
+    define('QR_CACHEABLE', true);       // use cache - more disk reads but less CPU power, masks and format templates are stored there
+    define('QR_CACHE_DIR', '/var/cache/php/phpqrcode/');       // used when QR_CACHEABLE === true
     define('QR_LOG_DIR', false);         // default error logs dir   
     
     define('QR_FIND_BEST_MASK', true);                                                          // if true, estimates best mask (spec. default, but extremally slow; set to false to significant performance boost but (propably) worst quality code
